polaris-service/build.gradle.kts

```diff
@@ -41,6 +41,11 @@ dependencies {
     exclude("ch.qos.reload4j", "reload4j")
     exclude("log4j", "log4j")
     exclude("org.apache.zookeeper", "zookeeper")
+    exclude("org.apache.hadoop.thirdparty", "hadoop-shaded-protobuf_3_25")
```

Comment:
Hadoop's an Apache project, why these excludes?

Reply:
I excluded because it's not actually used in Polaris.